src/extension.ts

```diff
@@ -282,7 +282,7 @@ async function activateServerForFolder(context: ExtensionContext, uri: Uri, fold
   };
 
   // Create the LSP client.
-  const langClient = new LanguageClient(langName, langName, serverOptions, clientOptions);
+  const langClient = new LanguageClient('haskell', langName, serverOptions, clientOptions);
```

Comment:
Everywhere else, we are prefering `langName` over hardcoding `'haskell'`, any reason to change it here?

Reply:
The LanguageClient uses this id parameter in `workspace.getConfiguration(id).trace.server` under the hood. And as far as I can tell `id` is only used for `getConfiguration`, so I think it should be safe to change. (Kind of weird that LanguageClient reads the config directly, but I guess its a way to standardize across language servers).